crates/celestia/celestia-integration-tests/tests/bootstrap.rs

```diff
@@ -31,7 +36,7 @@ fn test_celestia_bootstrap() -> Result<(), Error> {
         cosmos_builder: builder.clone(),
         chain_store_dir: store_dir.join("chains"),
         bridge_store_dir: store_dir.join("bridges"),
-        dynamic_gas: Some(DynamicGasConfig::default()),
```

Comment:
For CelestiaBootstrap, since this is chain-specific, we can probably make it either always have dynamic gas or not.